velox/functions/lib/aggregates/tests/utils/AggregationTestBase.h

```diff
@@ -69,7 +69,8 @@ class AggregationTestBase : public exec::test::OperatorTestBase {
       const std::vector<std::string>& aggregates,
       const std::string& duckDbSql,
       const std::unordered_map<std::string, std::string>& config = {},
-      bool testWithTableScan = true);
+      bool testWithTableScan = true,
+      bool testStreamingAggregation = false);
```

Comment:
This pattern of adding more and more optional parameters is problematic and it makes it difficult to use the API and read the calling code. I think it was a mistake to add testWithTableScan flag.

Can we use the existing allowInputShuffle_ flag to decide whether orderby + streaming agg should produce the same result as hash agg?

Reply:
@mbasmanova Hi Masha, it works, thanks for your guidance. I've updated the PR and rerun the `ReduceAggTest`, could you please take another look?

Reply:
>I think it was a mistake to add testWithTableScan flag.

Should we remove this flag and add `allowTableScan_` and `allowTableScan` API in `AggregationTestBase`?

Reply:
The question is what is so special about table scan that some aggregate functions don't support it. TableScan doesn't change the order of inputs, so it shouldn't really matter whether data comes from Values or TableScan.